db/migrations/versions/cd404dd517c5_.py

```diff
@@ -0,0 +1,59 @@
+"""empty message
+
+Revision ID: cd404dd517c5
+Revises: 4dd55a3dc51e
+Create Date: 2022-07-13 09:12:00.980796
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "cd404dd517c5"
+down_revision = "4dd55a3dc51e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "sub_criteria",
+        sa.Column("sub_criteria_id", sa.Text(), nullable=False),
```

Comment:
Should we use UUID for this too? Also, can we just call these fields `id` as its implied by the table name rather than `sub_criteria_id`?

Reply:
have changed this and updated methods and tests to reflect this 👍🏾